package.json

```diff
@@ -127,6 +127,7 @@
     "nodemon": "^2.0.20",
     "prettier": "2.8.8",
     "react-intersection-observer": "^9.5.2",
+    "request": "^2.88.2",
```

Comment:
👀?
This sneak in again?

Reply:
It turns out we do need it - I opened a separate PR. I don't know why it's not listed as a dependency, NextJS fails trying to build it: https://github.com/mozilla/blurts-server/pull/3417

Reply:
Now I'm 2&times; confused, since #3417 adds a [not very popular] [**requests**](https://www.npmjs.com/package/requests) (plural) instead of a [very popular, but deprecated, and generally vulnerable] [**request**](https://www.npmjs.com/package/request) (singular) dependency:

PACKAGE@VERSION | WEEKLY<br>DOWNLOADS | LAST<br>PUBLISH | NOTES
----|----:|----|----
request@2.88.2 | 10,914,14 | 4 years ago | DEPRECATED
requests@0.3.0 | 3,404 | 4 years ago